oozie-to-airflow/utils/el_utils.py

```diff
@@ -143,7 +143,8 @@ def convert_el_to_jinja(oozie_el, quote=True):
         parsed_fns = parse_el_func(jinjafied_el, EL_FUNCTIONS)
         return template.render(str=subbed_str, els=parsed_fns)
 
-    return '\'' + jinjafied_el + '\'' if quote else jinjafied_el
+    # This takes care of any escaping that must be done for python to understand
+    return repr(jinjafied_el) if quote else repr(jinjafied_el)[1:-1]
```

Comment:
Can we add some test cases. This change should be independent of the doc update.